tools/ld-disc-stacker/main.cpp

```diff
@@ -124,6 +137,28 @@ int main(int argc, char *argv[])
     bool noDiffDod = parser.isSet(noDiffDodOption);
     bool noMap = parser.isSet(noMapOption);
     bool passThrough = parser.isSet(passthroughOption);
+	
+    // Get the arguments from the parser
+	qint32 mode = 2;
```

Comment:
Indentation on these if() blocks is wrong.

Reply:
Still wrong - please convert the tabs here to 4 spaces.